pkg/trace/api/profiles_test.go

```diff
@@ -153,11 +153,21 @@ func TestProfilingEndpoints(t *testing.T) {
 }
 
 func TestProfileProxyHandler(t *testing.T) {
+	info.Version = "6.0.0"
```

Comment:
I agree that this is generally a bad an racy pattern for tests, which we can resolve in a separate PR, but regardless, it's best practice we set this back at the end of the test to whatever it was before the change.